src/services/codefixes/convertTypedefToType.ts

```diff
@@ -48,35 +50,63 @@ registerCodeFix({
             ];
         }
     },
-    getAllCodeActions: context => codeFixAll(context, errorCodes, (changes, diag) => {
-        const node = getTokenAtPosition(diag.file, diag.start);
-        if (node) doChange(changes, node, diag.file);
-    })
+    getAllCodeActions: context => codeFixAll(
+        context,
+        errorCodes,
+        (changes, diag) => {
+            const node = getTokenAtPosition(diag.file, diag.start);
+            if (node) doChange(changes, node, diag.file, context);
+        }
+    )
 });
 
-function doChange(changes: textChanges.ChangeTracker, node: Node, sourceFile: SourceFile) {
+function doChange(
+    changes: textChanges.ChangeTracker,
+    node: Node, sourceFile: SourceFile,
+    context: textChanges.TextChangesContext

```

Comment:
This is a nit, but looking at other fixes, it seems like `context` canonically comes first.